exec/src/coqnum.mli

```diff
@@ -0,0 +1,3 @@
+val int_of_coqpositive: Constr.constr -> int
+
```

Comment:
This empty line is inconsistent with the mli file for booleans.

Reply:
It is (or more precisely, it is consistent with the Coqstr module, the pattern of having two pairs of functions does not rise in Coqbool), but it is incomplete. I will add the missing functions in this file and add a dummy implementation (eg. raise ToBeImplented).